### PR TITLE
gh-110383: Fix documentation profile cumtime fix

### PR DESCRIPTION
**Pull request by aptakhin**
Rephrased for a clearer explanation: profiler docs inaccuracy by 2dfkit

https://mail.python.org/archives/list/docs@python.org/thread/DJNXVVP2UW4R6OAPRA6TE5WMHDC4NKZ6/#DJNXVVP2UW4R6OAPRA6TE5WMHDC4NKZ6

* Issue: gh-110383

----
:books: Documentation preview :books:: https://cpython-previews--112221.org.readthedocs.build/

### CODE REVIEW

File: Doc/library/profile.rst [hunk 1]
```diff
@@ -80,10 +80,10 @@ the following::
         1    0.000    0.000    0.000    0.000 _compiler.py:598(_code)
         1    0.000    0.000    0.000    0.000 _parser.py:435(_parse_sub)
 
-The first line indicates that 214 calls were monitored.  Of those calls, 207
+The first line indicates that 214 calls were monitored. Of those calls, 207
```
Comment (hugovk) [inline Doc/library/profile.rst]: Let's not change unrelated whitespace.
```suggestion
The first line indicates that 214 calls were monitored.  Of those calls, 207
```

File: Doc/library/profile.rst [hunk 2]
```diff
@@ -80,10 +80,10 @@ the following::
...
 were :dfn:`primitive`, meaning that the call was not induced via recursion. The
-next line: ``Ordered by: cumulative time``, indicates that the text string in the
-far right column was used to sort the output. The column headings include:
+next line: ``Ordered by: cumulative time`` indicates rows sorting order
+for the table below. The column headings include:
```
Comment (hugovk) [inline Doc/library/profile.rst]: The table is above, not below. Maybe reword like this?

```suggestion
next line: ``Ordered by: cumulative time`` indicates the output is sorted
by the ``cumtime`` values. The column headings include:
```

File: Doc/library/profile.rst (hunk 1 above)
Comment (aptakhin) [reply Doc/library/profile.rst]: Sure, I also thought it was unrelated, just pretty close to context too. Returned

Comment (hugovk) [reply Doc/library/profile.rst]: Thanks. I also prefer one space, but we allow either one or two: https://devguide.python.org/documentation/style-guide/#use-of-whitespace

**Comment by hugovk** [comment]
Thank you!

**Comment by aptakhin** [comment]
Thank you for the help!
